Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3 as base
+
+RUN pip install poetry
+COPY . /app
+WORKDIR /app
+RUN poetry install
```

Minor, but worth noting that we might choose to separate out the COPY command into two parts - first copying in the files that specify our dependencies that are directly relevant to `poetry install`, and then everything else, so that in future builds as often as possible we can cache the install step which is typically the slowest part of our builds
```suggestion
COPY pyproject.toml poetry.lock /app
WORKDIR /app
RUN poetry install
COPY . /app
```